Revisit page meta descriptions in Core docs - Closes #738 (#753)
LiskHQ/lisk-docs: modules/ROOT/pages/management/docker.adoc

= Docker image commands
Mona Bärenfänger <mona@lightcurve.io>

> :description: All Docker image commands relevant to manage Lisk Core with a Docker image.

Reviewer: ```suggestion
:description: How to manage Lisk Core with Docker.
```
